jbmc/src/java_bytecode/java_string_library_preprocess.cpp

```diff
@@ -322,23 +324,24 @@ java_string_library_preprocesst::convert_exprt_to_string_exprt(
 /// for char array references.
 /// \param operands: a list of expressions
 /// \param loc: location in the source
+/// \param function_name: name of the function in which the code will be added
 /// \param symbol_table: symbol table
 /// \param init_code: code block, in which declaration of some arguments may be
 ///   added
 /// \return a list of expressions
 exprt::operandst java_string_library_preprocesst::process_operands(
   const exprt::operandst &operands,
   const source_locationt &loc,
+  const irep_idt &function_name,
```

Comment:
Sometimes it's `function_name`, sometimes it's `function_id`. Can we only use the latter?